.gitignore

```diff
@@ -130,3 +130,6 @@ site
 
 # Local configuration files
 *.local.yaml
+
+# Sensitive credentials
+*-credentials.yaml
```

Comment:
This will only apply to users in this repo. We need to add this change to the `create-app` template as well, with some clear instructions for people to add it to existing repos, as it would be a manual change.

Reply:
You're right, I have also added to the `create-app` template :) 

Is it enough to update the changeset with instructions or is there more places where it should be communicated? 

We could also add some extra output to `create-github-app` that reminds the user that it's sensitive.